### PR TITLE
Remove starter code

### PR DESCRIPTION
**Pull request by draymond63**
## Implementation Description
Removed all mention of "entity" endpoints and uses. Please go through and make sure I didn't delete any files that might be necessary.

### CODE REVIEW

File: frontend/src/utils/FileUtils.ts [hunk 1]
```diff
@@ -1,9 +0,0 @@
-// eslint-disable-next-line import/prefer-default-export
```
Comment (jfdoming) [inline frontend/src/utils/FileUtils.ts:1]: Can you double check this file is not used? Just thinking about CSV downloads.

**Review by jfdoming — CHANGES_REQUESTED** [review]
Could you run the linter for these changes? Also, I'm wondering if there are libraries we can remove as well (could be beneficial in the sense of reducing bundle size).

File: frontend/src/utils/FileUtils.ts (hunk 1 above)
Comment (draymond63) [reply frontend/src/utils/FileUtils.ts:1]: Searching for `downloadFile` in the code, I couldn't find any example of it. There's a whole `CSVUtils` file with the function `generateCSV` that handles CSV download.

**Comment by draymond63** [comment]
> Could you run the linter for these changes?

I just did and there didn't seem to be any problems or changes made by the linter. Did you notice anything wrong?

File: frontend/src/utils/FileUtils.ts (hunk 1 above)
Comment (jfdoming) [reply frontend/src/utils/FileUtils.ts:1]: Great, thanks!

**Comment by jfdoming** [comment]
> > Could you run the linter for these changes?
> 
> I just did and there didn't seem to be any problems or changes made by the linter. Did you notice anything wrong?

I see a comment on the PR in App.tsx, but maybe it's stale.

**Review by jfdoming — APPROVED** [review]

**Pull request merged**